src/test/test_rebind.py

```diff
@@ -88,6 +88,14 @@ def pick_random_port():
 
 control_socket.sendall('SIGNAL HALT\r\n'.encode('utf8'))
 
-time.sleep(0.1)
+wait_for_log('exiting cleanly')
 print('OK')
-tor_process.terminate()
+
+try:
+    tor_process.terminate()
+except OSError as e:
+    if e.errno == 3: # No such process
```

Comment:
This should probably use the symbolic name from the `errno` module instead of hard coding a number.